From: Isaac Hermida <isaac.hermida@digi.com>
Date: Tue, 25 Feb 2025 12:32:45 +0000
Subject: [PATCH] use local copy of ssd_mobilenet_v1_quant model

This model is broken, and the Vela conversion is failing.
Until it is fixed, use a locally working copy.

Signed-off-by: Isaac Hermida <isaac.hermida@digi.com>
---
 download_models.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/download_models.py b/download_models.py
index ba1cce726e25..411538a19662 100644
--- a/download_models.py
+++ b/download_models.py
@@ -67,9 +67,9 @@ def download_all_models(model_dir, vela_dir):
 
     #Download object detection model
     #https://www.tensorflow.org/
-    url = 'https://storage.googleapis.com/tfhub-lite-models/tensorflow/lite-model/ssd_mobilenet_v1/1/metadata/2.tflite'
-    path = os.path.join(model_dir, 'ssd_mobilenet_v1_quant.tflite')
-    download_file('object detection', url, path)
+    # url = 'https://storage.googleapis.com/tfhub-lite-models/tensorflow/lite-model/ssd_mobilenet_v1/1/metadata/2.tflite'
+    # path = os.path.join(model_dir, 'ssd_mobilenet_v1_quant.tflite')
+    # download_file('object detection', url, path)
 
     #Download image classification model
     #https://www.tensorflow.org/
